### PR TITLE
UF-5363: Add testcontainers in IT-tests and use dept44 feign client

### PR DESCRIPTION
**Pull request by ronnberg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,32 +66,67 @@
...
             <groupId>org.testcontainers</groupId>
-            <artifactId>mariadb</artifactId>
+            <artifactId>junit-jupiter</artifactId>
             <version>${testcontainers.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.camunda.bpm</groupId>
+            <artifactId>camunda-engine</artifactId>
+            <version>7.18.0</version>
```
Comment (ronnberg) [inline pom.xml:83]: ${camunda-engine.version}

File: src/main/java/se/sundsvall/workflow/service/ProcessService.java [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
-import static se.sundsvall.workflow.integration.camunda.mapper.CamundaMapper.toCorrelationMessageDto;
-import static se.sundsvall.workflow.integration.camunda.mapper.CamundaMapper.toStartProcessInstanceDto;
+//import static se.sundsvall.workflow.integration.camunda.mapper.CamundaMapper.toCorrelationMessageDto;
+//import static se.sundsvall.workflow.integration.camunda.mapper.CamundaMapper.toStartProcessInstanceDto;
 
+import generated.se.sundsvall.camunda.StartProcessInstanceDto;
+import generated.se.sundsvall.camunda.VariableValueDto;
+import org.camunda.bpm.engine.RuntimeService;
+/*
+import org.camunda.bpm.engine.variable.type.ValueType;
```
Comment (ronnberg) [inline src/main/java/se/sundsvall/workflow/service/ProcessService.java:12]: Fix class

File: src/main/java/se/sundsvall/workflow/businesslogic/worker/MyWorker.java [hunk 3]
```diff
@@ -14,28 +11,28 @@
...
 
-import se.sundsvall.workflow.integration.camunda.handler.FailureHandler;
+import se.sundsvall.workflow.businesslogic.handler.FailureHandler;
 
 @Component
-@ExternalTaskSubscription("printMessage")
-public class PrintMessageWorker implements ExternalTaskHandler {
-	private static final Logger LOGGER = LoggerFactory.getLogger(PrintMessageWorker.class);
+@ExternalTaskSubscription("myWorker")
+public class MyWorker implements ExternalTaskHandler {
```
Comment (MickeWallner) [inline src/main/java/se/sundsvall/workflow/businesslogic/worker/MyWorker.java:18]: Extend AbstractWorker?

**Review by MickeWallner — APPROVED** [review]

File: src/main/java/se/sundsvall/workflow/businesslogic/worker/MyWorker.java (hunk 3 above)
Comment (josc04) [reply src/main/java/se/sundsvall/workflow/businesslogic/worker/MyWorker.java:18]: There is no need as the AbstractWorker only implements the method clearUpdateAvailable (which is not used in this worker).

File: src/main/java/se/sundsvall/workflow/api/ProcessResource.java [hunk 4]
```diff
@@ -19,44 +21,47 @@
...
 	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	public ResponseEntity<Void> updateProcess(
-		@Parameter(name = "businessKey") @PathVariable final String businessKey,
-		@Parameter(name = "phase") @PathVariable final String phase) {
+		@Parameter(name = "processInstanceId") @PathVariable final String processInstanceId) {
 
-		service.updateProcess(businessKey, TENANTID_TEMPLATE, phase);
-		return new ResponseEntity<>(HttpStatus.OK);
+		LOGGER.info("Request for update of process instance with id {} has been received", processInstanceId);
```
[the remark this answers is not in the document]
Comment (josc04) [reply src/main/java/se/sundsvall/workflow/api/ProcessResource.java:62]: @sonatype-lift ignore

**Pull request merged**